lib/Raven/ErrorHandler.php

```diff
@@ -138,6 +138,8 @@ public function handleFatalError()
                 @$error['message'], 0, @$error['type'],
                 @$error['file'], @$error['line']
             );
+            
+            $this->client->useCompression &= PHP_VERSION_ID > 70000;
```

Comment:
Maybe it's just me, but using bitwise operator on boolean feels wrong to me :)

Reply:
Wops! That works by mere coincidence I think :sweat_smile: fixing now!